packages/amplication-client/src/Components/EditableTitleField.tsx

```diff
@@ -1,9 +1,9 @@
 import React from "react";
-import { TextField, Props } from "@amplication/design-system";
+import { TextField, TextFieldProps } from "@amplication/design-system";
```

Comment:
Why did you rename `Props ` to `TextFieldProps`? I don't believe this is essential for the Nx migration.

Our repo has **a lot of traffic** and changes like this can easily cause a bunch of conflicts.